cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js

```diff
@@ -220,6 +219,7 @@ export default class ColumnActionsDropdown extends Component {
   }
 
   componentWillUnmount() {
+    this.eventEmitter.off('CLOSE_POPOVER', this.toggleDropdown.bind(this, false));
```

Comment:
what's the reason for this change? this can cause multiple dropdown to open, right?

Reply:
Actually its not. The `CLOSE_POPOVER` event is emitted only when we `execute` any directive. The `this.documentClick$` in line 249 closes the popover if the user clicks anywhere else in the document. I am not sure why we were listening to this event in the `constructor` and switching it off in `componentWillMount`. But moving it to `componentWillUnmount` didn't change and the popover closes as it used to be. 

There is an inherent JS error that is happening with popover that I was trying to debug. Couldn't get to it in time. Will see if I can fix the js error.